scenarios/signalr.benchmarks.yml

```diff
@@ -53,7 +53,7 @@ profiles:
           - http://asp-perf-load:5001
   aspnet-perf-win:
     variables:
-      serverAddress: 10.0.0.104
+      serverAddress: 10.0.0.110
```

Comment:
Is this for local runs only?

Reply:
What do you mean?

Reply:
Does the CI use this and was just broken?

Reply:
Yea the CI doesn't use this, these are the other machines used for test. The IP was just wrong.